### PR TITLE
Adding some efiling tests

### PR DESCRIPTION
**Pull request by LindsayYoung**
No idea why the sort test does not return reciept_date

### CODE REVIEW

**Comment by LindsayYoung** [comment]
closes #1880

File: tests/test_filings.py [hunk 1]
```diff
@@ -120,3 +120,57 @@ def test_regex(self):
...
+class TestEfileFiles(ApiBaseTest):
+
+    def test_filter_date_efile(self):
+        [
+            factories.EFilingsFactory(committee_id='C010', beginning_image_number=2, receipt_date=datetime.date(2012, 1, 1)),
+            factories.EFilingsFactory(committee_id='C011', beginning_image_number=3, receipt_date=datetime.date(2013, 1, 1)),
+            factories.EFilingsFactory(committee_id='C012', beginning_image_number=4, receipt_date=datetime.date(2014, 1, 1)),
+            factories.EFilingsFactory(committee_id='C013', beginning_image_number=5, receipt_date=datetime.date(2015, 1, 1)),
+        ]
+        # print(factories.EFilingsFactory(committee_id='C013', receipt_date=datetime.date(2015, 1, 1)).__dict__)
```
Comment (ccostino) [inline tests/test_filings.py]: There's one a couple of lines below too.

**Comment by ccostino** [comment]
@LindsayYoung I only had a couple of nit-pick things on this; if all is good to go I say we merge this so it doesn't sit any longer than it has. :-)

**Comment by LindsayYoung** [comment]
Thanks for pointing those out! I will do that now

**Comment by LindsayYoung** [comment]
@ccostino should be good to go- thanks!

**Pull request merged**